apps/main/src/features/add-gauge-reward-token/ui/AddRewardToken.tsx

```diff
@@ -10,11 +10,12 @@ import TxInfoBar from '@/ui/TxInfoBar'
 
 import { useSignerAddress } from '@/entities/signer'
 import { DistributorInput, EstimatedGasInfo, FormActions, TokenSelector } from '@/features/add-gauge-reward-token/ui'
-import { formDefaultOptions } from '@/shared/model/form'
-import { FormErrorsDisplay } from 'shared/ui/forms'
+import { formDefaultOptions } from '@/shared/curve-lib'
+import { FormErrorsDisplay } from '@/ui/FormErrorsDisplay'
```

Comment:
```suggestion
import { FormErrorsDisplay } from '@shared/ui-kit/FormErrorsDisplay'
```
Try doing these imports via FSD.

Reply:
This seems a bit unrelated to this PR. We could just replace `@/ui` with `@/shared/ui`